spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -269,25 +270,24 @@ but with the addition of an external configuration file containing
 overrides to annotations. The additional configuration file allows an
 administrator to customize the behavior or configuration of the Web
 Service – such as the endpoint URL - without changing the Java source.
-* Generation of Java EE 5 Web Services - In this model, a tool uses the
-metadata in +
-the annotated Java class to generate a Java EE 5 Web Service based on
-JSR-109 and JAX-WS. The initial Web Service is generated from the
+* Generation of Jakarta Web Services - In this model, a tool uses the
+metadata in the annotated Java class to generate a Jakarta Web Service based on
+Jakarta Enterprise Web Services and Jakarta XML Web Services. The initial Web Service is generated from the
```

Comment:
to generate a Jakarta **EE** Web Service based on Jakarta Enterprise Web Services and Jakarta XML Web Services **Specifications**.

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/0b6c04287b2400cd4ccaeb1cd387e9f5c3535f1c